src/dc_stock.rs

```diff
@@ -1,338 +1,112 @@
-use crate::constants::Event;
```

Comment:
Please rename to `stock.rs` at the end to indicate that the conversion was done.